CUSTOM_BUNDLE.md

```diff
@@ -55,26 +55,20 @@ npm run custom-bundle -- --traces scatter,scattergl --strict
 ```
 
 Use the `out` option to change the bundle filename (default `custom`).
-The new bundle will be created in the `dist/` directory and named `plotly-<out>.min.js` or `plotly-<out>.js` if unminified.
+The new bundle will be created in the `dist/` directory and named `plotly-<out>.js` and `plotly-<out>.min.js` for minified.
 ```sh
 npm run custom-bundle -- --out myBundleName
 ```
 
-Use the `unminified` option to disable compression.
-```sh
-npm run custom-bundle -- --unminified
-```
-
 # Example illustrating use of different options together
-To create an unminified custom bundle named `myScatters` including `scatter`, `scattergl` and `scatter3d` traces without any transforms:
+To create a custom bundle named `myScatters` including `scatter`, `scattergl` and `scatter3d` traces without any transforms:
 ```sh
 npm run custom-bundle -- \
-    --unminified \
     --out myScatters \
     --traces scatter,scattergl,scatter3d \
     --transforms none
 ```
 Or simply on one line:
 ```sh
-npm run custom-bundle -- --unminified --out myScatters --traces scatter,scattergl,scatter3d --transforms none
+npm run custom-bundle -- --out myScatters --traces scatter,scattergl,scatter3d --transforms none
```

Comment:
I'm not against removing unminified here, but could you provide some context on why that's in this PR?

Reply:
Good question. I need to change centralized partial bundle script to bring back these options. Having only minified bundle could save users few seconds. So I'll try to revert this.

Reply:
Reverted and addressed in [8bb92f8](https://github.com/plotly/plotly.js/pull/6909/commits/8bb92f8c02d7d3855dbf2c2d161a9e265d8e2d96).